packages/gatsby/src/recipes/index.js

```diff
@@ -0,0 +1,5 @@
+const path = require(`path`)
+module.exports = recipe => {
+  const cli = require(`import-jsx`)(require.resolve(`./cli`))
```

Comment:
This should be compiled on build time. If you'd put this in gatsby-cli it would have already been setup for you.